Sources/ContainerCommands/BuildCommand.swift

```diff
@@ -207,7 +207,33 @@ extension Application {
                     buildFilePath = resolvedPath
                 }
 
-                let buildFileData = try Data(contentsOf: URL(filePath: buildFilePath))
+                let buildFileData: Data
```

Comment:
Looks good to me.

Could you add a test for this? Just copy one of the simple tests in CLIBuilderTest and supply the Dockerfile via stdin instead of a file.

I think this will require creating a separate helper function to  https://github.com/apple/container/blob/main/Tests/CLITests/Subcommands/Build/CLIBuildBase.swift#L112, since there's nothing in that test support stuff that knows how to pipe stdin into `container build`.